libtesting-utils/build.gradle

```diff
@@ -24,4 +24,5 @@ dependencies {
     implementation dependenciesList.kotlinReflect
     implementation dependenciesList.mapboxCommonNative
     implementation dependenciesList.mockk
+    implementation 'org.json:json:20210307'
```

Comment:
what is it? 🤔

Reply:
This is needed for the Java JSON library to be available in the test builds that are run without robolectric because, as far as I understand, the Java JSON lib is imported by the Android SDK.

The new tests were crashing with missing JSON implementation without this dependency.